app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RelationshipsController < ApplicationController
+  before_action :require_user
+
+  def index
+    @relationships = current_user.following_relationships
+  end
+
+  def destroy
+    relationship = Relationship.find(params[:id])
+    relationship.destroy if relationship.follower == current_user
+    redirect_to people_path
+  end
+
+  def create
+    leader = User.find(params[:leader_id])
+    if current_user.can_follow?(leader)
+      Relationship.create(leader_id: params[:leader_id], follower: current_user) 
```

Comment:
You already have `leader` object present at this stage. Rather than `leader_id: params[:leader_id]` you could do `leader: leader` which looks more convincing.